requirements.txt

```diff
@@ -30,7 +30,6 @@ itsdangerous==0.24        # via flask
 jinja2==2.10
 jsonschema==2.6.0
 markupsafe==1.0           # via jinja2
-path.py==11.5.0
```

Comment:
Do we not need path.py anymore? Did this leak over from python 3 only branch?

Reply:
yeah...